dask/dataframe/tests/test_dataframe.py

```diff
@@ -5862,6 +5862,22 @@ def test_is_monotonic_deprecated():
     assert_eq(expected, result)
 
 
+def test_monotonic():
```

Comment:
[non-blocking comment] Just wanted to note that this test also passes on `main`, so doesn't appear to be sensitive to the "returns `np.bool_`" point we're avoiding. Just mentioning in case your intention was to test that.

Reply:
maybe this depends on the pandas version? This test triggered it over in dask-expr and we're using the same basic code over there.